sys-libs/mesa/patches/mesa-21.0.3.patchset

```diff
@@ -0,0 +1,28 @@
+From f304616287291bcc919f8502019a440239c7f353 Mon Sep 17 00:00:00 2001
+From: Ken Mays <kmays2000@gmail.com>
+Date: Fri, 16 Jul 2021 06:34:04 -0200
+Subject: util/u_thread.h: Redefined ALIGN for Haiku
+
+
+diff --git a/src/util/u_thread.h b/src/util/u_thread.h
+index c5f7fb6..cfcd6aa 100644
+--- a/src/util/u_thread.h
++++ b/src/util/u_thread.h
+@@ -60,6 +60,14 @@
+ #define cpu_set_t cpuset_t
+ #endif
+ 
++#ifdef __HAIKU__
++/* pthread_np.h -> sys/param.h -> machine/param.h
++ * - defines ALIGN which clashes with our ALIGN
++ */
++#undef ALIGN
```

Comment:
Wait, which ALIGN is this using, then? Mesa's or ours? Mesa's should be the one used...

Reply:
You can't just close comments without addressing them.